pyproject.toml

```diff
@@ -2,25 +2,32 @@
 requires = ["flit_core >=2,<4"]
 build-backend = "flit_core.buildapi"
 
-[tool.flit.metadata]
-module = "pipewire_python"
+[project]
+name = "pipewire_python"
 author = "Pablo Diaz"
```

Comment:
I think its time to change to pipewire_python's authors

Reply:
Fair point! I've updated to the listed dictionaries style that seems available for Flit and Poetry.

@annaabsi, @kwint, @mrteathyme: I presently have you listed as authors for your contributions. But I'd like to confirm your willingness to be listed here (don't want to put your names "out there" if you don't feel comfortable with it).

Reply:
maybe only "pipewire_python's authors"

Reply:
Thanks for asking, I rather not be listed as an author. I'm fine with being listed as a contributor, but author sounds like to much recognition for my 5 lines of code and I don't think I will add much to this repo in the future